### ISSUE TITLE
Uninitialized constant Friends::Activity::Set when graphing by tag

### ISSUE THREAD

**Issue by andypearson** [JacobEvelyn/friends#133]
Just spotted this (because I only just tagged all of my friends!)

```
$ friends --debug graph --tagged "#friday"
/Users/andypearson/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/friends-0.23/lib/friends/activity.rb:139:in `hashtags': uninitialized constant Friends::Activity::Set (NameError)
        from /Users/andypearson/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/friends-0.23/lib/friends/activity.rb:134:in `includes_hashtag?'
        from /Users/andypearson/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/friends-0.23/lib/friends/introvert.rb:493:in `block in filtered_activities'
        from /Users/andypearson/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/friends-0.23/lib/friends/introvert.rb:493:in `select'
        from /Users/andypearson/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/friends-0.23/lib/friends/introvert.rb:493:in `filtered_activities'
        from /Users/andypearson/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/friends-0.23/lib/friends/introvert.rb:313:in `graph'
        from /Users/andypearson/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/friends-0.23/bin/friends:352:in `block (2 levels) in <top (required)>'
        from /Users/andypearson/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/gli-2.13.4/lib/gli/command_support.rb:126:in `execute'
        from /Users/andypearson/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/gli-2.13.4/lib/gli/app_support.rb:296:in `block in call_command'
        from /Users/andypearson/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/gli-2.13.4/lib/gli/app_support.rb:309:in `call_command'
        from /Users/andypearson/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/gli-2.13.4/lib/gli/app_support.rb:83:in `run'
        from /Users/andypearson/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/friends-0.23/bin/friends:465:in `<top (required)>'
        from /Users/andypearson/.rbenv/versions/2.3.0/bin/friends:23:in `load'
        from /Users/andypearson/.rbenv/versions/2.3.0/bin/friends:23:in `<main>'
```

**Comment by JacobEvelyn** [comment]
Good catch! I'll take a look.

**Comment by JacobEvelyn** [comment]
I'm not sure why this didn't happen in development (possibly something to do with the loading of development dependencies?) but I'm fixing this now. Note that this is another small vote in favor of more testing, though it would be easy to write integration tests that still didn't catch this (and possibly hard to write integration tests that *did* catch this, depending on where the underlying issue is).

**Comment by JacobEvelyn** [comment]
This is fixed in version 0.24, which just released.
